postgresql/helpers.go

```diff
@@ -267,6 +267,34 @@ func validatePrivileges(d *schema.ResourceData) error {
 	return nil
 }
 
+func arePrivilegesEqual(granted *schema.Set, wanted *schema.Set, d *schema.ResourceData) bool {
```

Comment:
You seem to all read `wanted` from `d` that you also pass as function arguments, so I guess you could pass `d` only?

```suggestion
func resourcePrivilegesEqual(granted *schema.Set, d *schema.ResourceData) bool {
  wanted := d.Get("privileges").(*schema.Set)
```

Reply:
changed with 8dfcf905d1bfb5746001a6e98fca82492058bb1b